starport/cmd/generate_go.go

```diff
@@ -0,0 +1,36 @@
+package starportcmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/tendermint/starport/starport/pkg/clispinner"
+	"github.com/tendermint/starport/starport/services/chain"
+)
+
+func NewGenerateGo() *cobra.Command {
+	return &cobra.Command{
+		Use:   "go",
+		Short: "Generate proto based Go code needed for the app's source code",
```

Comment:
Is it the `protoc` command?

Reply:
This one is for generating Go code that goes under `types/`.